containers/daap-delete-data-product/tests/unit/mock_test.py

```diff
@@ -65,91 +75,144 @@ def test_success(
         self,
         s3_client,
         create_glue_tables,
-        create_failed_raw_and_curated_data,
+        create_fail_data,
+        create_raw_data,
+        create_curated_data,
         data_product_name,
         glue_client,
         event,
         fake_context,
     ):
-        # Assert we have the correct number of database tables created
-        tables = glue_client.get_tables(DatabaseName=data_product_name)
-        assert len(tables["TableList"]) == 3
+        # Call the handler
+        result = delete_data_product.handler(event, fake_context)
 
-        # Assert we have the correct number of metadata files to begin with
-        prefix = f"{data_product_name}/"
-        response = s3_client.list_objects_v2(
-            Bucket=os.getenv("METADATA_BUCKET"),
-            Prefix=prefix,
+        assert (
+            json.loads(result.get("body"))["message"]
+            == f"Successfully removed data product '{data_product_name}'."
         )
-        assert response.get("KeyCount") == 12
 
-        # Assert we have the correct number of fail files to begin with
+    def test_fail_files_are_deleted(
```

Comment:
```suggestion
     def test_files_in_fail_are_deleted(
```
suggest renaming as `test_fail_` can be misleading

Reply:
Makes sense